scatterer/Core.cs

```diff
@@ -19,19 +19,9 @@ public partial class Core: MonoBehaviour
 		
 		private Core()
 		{
-			if (instance == null)
-			{
-				instance = this;
-				Debug.Log("[Scatterer] Core instance created");
-			}
-			else
-			{
-				//destroy any duplicate instances that may be created by a duplicate install
-				Debug.Log("[Scatterer] Destroying duplicate instance, check your install for duplicate mod folders");
-				UnityEngine.Object.Destroy (this);
-			}
+			
```

Comment:
Any reason to leave the empty constructor?

Reply:
Probably not, I was just doing the minimum to fix the problem